src/App.js

```diff
@@ -1,33 +1,26 @@
 import React, { Component } from 'react';
 import './App.scss';
+import { SearchBar } from './components/SearchBar';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export class App extends Component {
-  state = {};
+  state = {
+    searchValue: '',
+  };
+
+  handleSearch = (e) => {
+    this.setState({ searchValue: e.target.value.toLowerCase() });
```

Comment:
переносы обязательны
```suggestion
    this.setState({
      searchValue: e.target.value.toLowerCase()
    });
```